lib/fs_utils/asset.js

```diff
@@ -66,4 +74,56 @@ class Asset {
   }
 }
 
+class CompiledAsset {
```

Comment:
Currently brunch has two notions of file: SourceFile, something that's compiled along with source maps and stuff and then joined into one file; and Asset, which is just copied as-is.

I couldn't find a place for a new type of asset, one that is compiled, in either of these so I created a new class with has the parts it needs from both (without the stuff it does *not* need)